entrypoint.sh

```diff
@@ -17,6 +17,12 @@ while :; do
     shift
     ;;
 
+  --g711-only-eu)
```

Comment:
could you rename this option to `--g711-only-alaw-preferred`

Reply:
Resolved